firebase/rules/database.rules.json

```diff
@@ -79,7 +79,7 @@
           },
           "tags": {
             "$key1": {
-              ".validate": "newData.isString()"
+              ".validate": "newData.isString() && newData.val().length <= 50"
```

Comment:
@cburgdorf this is what `String50[]` translates to.

Reply:
I *think* that is the right thing but would certainly be better to have tests prove it :) The reason I'm so keen on this stuff is because otherwise a malicious attacker can flood the system with db entries where each tag uses 10 MB of data (millions of characters). In fact, I guess it would be even better to not only limit the length of each tag but also the length of the array itself.